src/main/java/com/redhat/cloud/notifications/routers/models/SettingsValueJsonForm.java

```diff
@@ -60,12 +80,12 @@
                         default:
                             return;
                     }
-                    form.fields.add(field);
+                    fieldsField.fields.add(field);
                 });
             });
         });
 
-        return forms;
+        return form;
     }
 
     private static Field createLabelField(String label, List<Field> parentContainer) {
```

Comment:
Shouldn't this method be removed?

Reply:
That comment was meant for the new version of the file.

Reply:
good catch, looks like it is not used anymore.